### PR TITLE
update config with note for full list of support configs

### PR DESCRIPTION
**Pull request by drizuid**
closes https://github.com/linuxserver/docker-cops/issues/45

### CODE REVIEW

**Comment by mikespub** [comment]
Hi @drizuid since this is a PHP file that will be included, and we want to avoid sending any *blank spaces* or *new lines* by mistake, could you please insert the comment *after* the first 3 lines:
```
<?php
    if (!isset($config))
        $config = array();

```
Thanks

File: root/defaults/config_local.php [hunk 1]
```diff
@@ -1,3 +1,13 @@
+/*
+ ***************************************************
+ * Please read config_default.php for all possible
+ * configuration items.
+ * For changes in config_default.php see CHANGELOG.md
+ * on the upstream project site.
+ * https://github.com/mikespub-org/seblucas-cops/blob/main/config_default.php
+ ***************************************************
+ */
+
```
Comment (mikespub) [inline root/defaults/config_local.php:10]: Oops, this is a newline - we just messed up any reply from the web server here :-)

Please move the comment *below* the <?php part as mentioned in the comment

**Review by mikespub — CHANGES_REQUESTED** [review]

**Comment by drizuid** [comment]
oops, i should've caught that :D my bad. That is what i get for doing a PR from the phone!
LMK if this looks good now.

**Comment by mikespub** [comment]
hi @drizuid you're good to go. For some reason Github won't let me approve my own review

**Review by thelamer — APPROVED** [review]

**Pull request merged**